pkg/kubelet/rkt/rkt.go

```diff
@@ -1315,6 +1315,25 @@ func (r *Runtime) setupPodNetwork(pod *v1.Pod) (string, string, error) {
 	return netnsName, status.IP.String(), nil
 }
 
+// For a hostPath volume: rkt doesn't create any missing volume on the node/host so we need to create it
+func createHostPathVolumes(pod *v1.Pod) error {
+	for _, v := range pod.Spec.Volumes {
+		if v.VolumeSource.HostPath != nil {
+			_, err := os.Stat(v.HostPath.Path)
```

Comment:
@JulienBalestra I think this can be simplified by 
```go
err := os.Mkdir(v.HostPath.Path)
if err != nil && os.IsExist(err) {
   return err
}
```